src/com/lftechnology/java/training/alina/jdbc/constants/Constants.java

```diff
@@ -0,0 +1,33 @@
+package com.lftechnology.java.training.alina.jdbc.constants;
+
+/**
+ * Common constants
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ *
+ */
+public class Constants {
+
+    private Constants() {
+
+    }
+
+    public static final char ADMIN_EXIT = 'g';
+    public static final char EMPLOYEE_EXIT = 'e';
+    public static final char EMPLOYEE_EDIT_EXIT = 'f';
+    public static final boolean TERMINATED = true;
+    public static final boolean NOT_TERMINATED = false;
+    public static final boolean DELETED = true;
+    public static final boolean NOT_DELETED = false;
+    public static final String FULLNAME = "fullname";
+    public static final String DEPARTMENT = "department";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)